Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -98,34 +99,82 @@ const Contest = () => {
         [],
     );
 
-    return (
-        <>
-            <div className={styles.headingContest}>
-                <Heading
-                  type={HeadingType.primary}
-                  className={styles.contestHeading}
-                >
-                    {contest?.name}
-                </Heading>
-                <Heading type={HeadingType.secondary} className={secondaryHeadingClassName}>
-                    {renderTimeRemaining()}
-                    {renderScore()}
-                </Heading>
-            </div>
-
-            <div className={styles.contestWrapper}>
-                <div className={navigationContestClassName}>
-                    <ContestTasksNavigation />
+    const determineErrorMessage = useCallback(() => {
+        if (!validationError.contestIsFound) {
+            return `${contest?.name} - ${ContestValidationErrors.NotFound}`;
+        } if (!validationError.contestIsNotExpired) {
```

Comment:
Does this meet eslint checks?

Reply:
There were no errors in the IDE, but I don't remember running yarn run lint. Not sure how I did not notice how ugly this is, but I separated the ifs with new lines.